partition-alloc/CMakeLists.txt

```diff
@@ -65,37 +65,43 @@ set(PA_SRCS
 
 list(TRANSFORM PA_SRCS PREPEND ${EXTERNAL_DIR}/chromium/src/base/allocator/partition_allocator/)
 
-add_library(partition-alloc SHARED
+add_library(partition-alloc_unpadded SHARED
 	src/allocator_shim_default_dispatch_to_partition_alloc.cc
 	src/get_pkey.cc
 	src/shared_allocator.cc
     ${PA_SRCS})
 
+pad_tls_library(partition-alloc_unpadded partition-alloc)
```

Comment:
This creates a target named `partition-alloc-padding` but nginx tries to build a CMake target named `partition-alloc` which is why CI is failing. Since the `INPUT` parameter to `pad_tls_library` is usually `$X_unpadded` it might make sense to name the target for the padded lib exactly like the `OUTPUT` parameter instead of with the `-padding` suffix.

Reply:
actually I just noticed that `pad_tls_library` should create a target named `OUTPUT` so I'm not entirely sure what's going on here

Reply:
It also creates a library target of `${OUTPUT}`, so I'm not sure what the problem is exactly. I'll investigate.

Reply:
I think my issue was that I needed a separate target to use `add_library` because I can't `add_library` for a custom command without a custom target in between.

Reply:
I worked around this in 1ce7fba8f